stdlib/cache/src/main/ballerina/src/cache/cache.bal

```diff
@@ -28,15 +28,14 @@ import ballerina/time;
 #                            `maxAgeInSeconds` property set when inserting item to the cache
 # + cleanupIntervalInSeconds - Interval of the timer task, which will clean up the cache
 public type CacheConfig record {|
-    int capacity = 100;
+    int capacity = 20;
```

Comment:
@kalaiyarasiganeshalingam are we updating the default cache value with this patch?
@daneshk

Reply:
yes, we thought of changing it Chanaka. Do you see any issues? This is because HTTP cache also to 16 now